plugins/builds/triggers/helpers.js

```diff
@@ -700,11 +700,18 @@ function getSubsequentJobs(workflowGraph, startNode) {
         return [];
     }
 
+    console.log('edges: ', edges);
+
     const visiting = [start];
 
     const visited = new Set(visiting);
 
-    edges.forEach(edge => nodeToEdgeDestsMap[edge.src].push(edge.dest));
+    edges.forEach(edge => {
+        if (!nodeToEdgeDestsMap[edge.src]) {
+            nodeToEdgeDestsMap[edge.src] = [];
+        }
+        nodeToEdgeDestsMap[edge.src].push(edge.dest);
```

Comment:
This looks like a bug in the workflow graph. `edges` in the workflow graph needs to be fixed which building the workflow graph not to include `~` prefix in the `src`/`dest`.

Reply:
Per discussion, either the `src` or `dest` of the edge should be a job in the current pipeline.
Long term fix would be to find/fix why we have edges with neither `src` nor `dest` is part of the pipeline.

For short term, we can have this temporary fix. Can we add a TODO/note to come back to this?

Reply:
I put a `todo` https://github.com/screwdriver-cd/screwdriver/pull/3202/commits/aa29c13d7fe617672f742efb2380dfff8042dfd0